tests/ami/scripts/test_windows.ps1.j2

```diff
@@ -176,19 +176,6 @@ Log-Message ""
 Log-Message "native agent status output verified"
 Log-Message ""
 
-#
-# Verify build_info file exists
-#
-
-Log-Message ""
-Log-Message "Verifying build_info file exists"
```

Comment:
I guess we don't need this check anymore since agent will fail to start if the install info file doesn't exist, right?

Reply:
Yes, And in the case of frozen binaries (and Windows MSI has frozen binaries), the `install_info.json` file is "baked" to the binary itself.